python/feathub/processors/flink/tests/test_redis_client.py

```diff
@@ -50,6 +54,13 @@ def setUpClass(cls) -> None:
     def tearDownClass(cls) -> None:
         super().tearDownClass()
         cls.redis_container.stop()
+        # clean up the java_gateway so that it won't affect other tests.
+        with java_gateway._lock:
+            if java_gateway._gateway is not None:
```

Comment:
It seems inconsistent to explicitly tear down java_gateway here while we don't have code that explicitly setup  java_gateway in `setUpClass`.

Is this the expected way to use `pyflink.java_gateway` or is this due to bug? If it is due to bug, can you provide the JIRA ticket here and add TODO to remove this code snippet?